src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -68,6 +69,9 @@ class JenkinsJobManager {
             if (startOnCreate && missingJob.jobName.contains("Rev.com-featurebranch-build-feature")) {
```

Comment:
I am thinking it might be a good idea to add a new param for 
`Rev.com-featurebranch-build-feature` and for `Rev.com-featurebranch-deploy-feature` What do you think?